actors/builtin/storage_miner/storage_miner_actor_state.go

```diff
@@ -35,9 +35,6 @@ type MinerPoStState struct {
 	//  `epochUndefined` if not currently challeneged.
 	SurpriseChallengeEpoch abi.ChainEpoch
 
-	// Not empty iff the miner is challenged.
-	ChallengedSectors []abi.SectorNumber
```

Comment:
Blocking: unless I am missing something, dont think we should remove this even though a constant number is challenged. This is a snapshot of the sectors that are challenged. Let's discuss in sync

Reply:
I don't understand what this does and assumed that this was based on a misunderstanding of how the proofs lib works. All sectors are submitted to the proofs lib, which randomly samples a wanted number cc @porcuquine. What is this meant to do @zixuanzh ?

Reply:
synced w ZX. This is meant to be a cached version of the proving set used to know what got challenged (in case new sectors are added). Must be worked with concurrently with whole power accounting concept. But was wrong in actor code right now (or incomplete in any case). Handled in follow up PR.

Reply:
That makes sense, should be simple as a CID.